### DISCUSSION TITLE
Two GC9A01 displays on one esp32 with different contents

### DISCUSSION THREAD

**Discussion by griffinmt** [Bodmer/TFT_eSPI#2345] [Q&A - General]
Is it possible to attach all the pins of one display to a second display providing the CS pins are different? And if they are the same type of display, how do you invoke them with TFT_eSPI?
EG:
TFT_eSPI   tft1;
TFT_eSPI   tft2;
How do you tell each which CS to use?
Since they are using the same user_setup etc.

**Comment by Bodmer** [comment]
Control the cS lines in the sketch. See turorial here:
https://www.youtube.com/watch?v=cCgNHIHijhs
